tools/rng_test/rng_test.py

```diff
@@ -0,0 +1,135 @@
+###########################################################################
+#
+# Copyright 2023 Samsung Electronics All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+# either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+#
+###########################################################################
+
+from operator import eq
+
+import serial
+import time
+import threading
+import sys
+
+# ------------- Global Variables -------------
+port_num = sys.argv[1]          # port number (user input)
+baudrate = int(sys.argv[2])     # baudrate (user input)
+
+iteration = int(sys.argv[3])    # Number of iteration (user input)
+iter_cnt = 0                    # Current number of iteration
+
+isExit = False                  # True if exit the script
+total_results = {}              # Dictionary for storing total results
+rand_nums = []                  # list for saving generated random number
+
+
+def rxfromUART(port):
+    global isExit
+
+    prev = ''
+    while (isExit == False):
+        time.sleep(0.01)
+
+        line = port.readline().decode()
+        if (prev.find("Random : ") != -1):
+            rand_nums.append(line)
+        #sys.stdout.write("%s" % (line))
+        prev = line
+
+
+def txtoUART(port):
+    global isExit
+    global iter_cnt
```

Comment:
Lets follow similar naming convention
> iterCnt

Reply:
iterCnt and isExit are violations of TizenRT coding rules. Let's use is_exit and iter_cnt.